xt/lib/PageObject/App/Invoices/Line.pm

```diff
@@ -75,7 +75,7 @@ sub field_value {
                       \@name="$field_map{$label}_${id}"]});
     die "Invoice line column $field_map{$label}_${id} not found"
         if not defined $field;
-    my $rv = $field->value;
+    my $rv = $field->get_attribute('value');
```

Comment:
`$field->value` means to use the Dojo/HTML widget's `value()` method. Why change it to a more low-level method?

Reply:
Leftover. Restored.